diff --git a/node_modules/ton-core/dist/address/Address.js b/node_modules/ton-core/dist/address/Address.js
index 3d8c5dd..5fdce44 100644
--- a/node_modules/ton-core/dist/address/Address.js
+++ b/node_modules/ton-core/dist/address/Address.js
@@ -137,7 +137,6 @@ class Address {
         this[_a] = () => this.toString();
         this.workChain = workChain;
         this.hash = hash;
-        Object.freeze(this);
     }
     equals(src) {
         if (src.workChain !== this.workChain) {
diff --git a/node_modules/ton-core/src/address/Address.ts b/node_modules/ton-core/src/address/Address.ts
index c554e17..c3e33a9 100644
--- a/node_modules/ton-core/src/address/Address.ts
+++ b/node_modules/ton-core/src/address/Address.ts
@@ -112,7 +112,6 @@ export class Address {
     constructor(workChain: number, hash: Buffer) {
         this.workChain = workChain;
         this.hash = hash;
-        Object.freeze(this);
     }
 
     toRawString = () => {
